README.md

```diff
@@ -5,3 +5,114 @@ Arches for Science an Arches Application designed to help conservation scientist
 
 Please see the [project page](http://archesproject.org/) for more information on the Arches project.
 
+
+## Installation
+
+If you are installing Arches for Science for the first time, you can install it as an Arches application into a project or run it directly as an Arches project. Running Arches for Science as a project can provide some convienience if you are a developer contributing to the Arches for Science project. Otherwise, **we strongly recommend running Arches for Science as an Arches Application** within a project because that will allow greater flexibility in customizing your project without the risk of conflicts when upgrading to the next version of Arches for Science.  
+
+
+### If installing for development
+Clone the arches-for-science repo and checkout the latest `dev/x.x.x` branch. 
+Navigate to the `arches-for-science` directory from your terminal and run:
+```
+pip install -e .
+```
+
+### If installing for deployment, run:
+```
+pip install arches-for-science
+```
+
+`Important`: Installing the arches-for-science app will install Arches as a dependency. This may replace your current install of Arches with a version from PyPi. If you've installed Arches for development using the `--editable` flag, you'll need to re pip install arches after installing arches-for-science
+
+## Project Configuration
```

Comment:
This seems mostly geared toward development.  Perhaps in another iteration we can make some documentation for a production release.

Reply:
Good point. I've moved that text up to the 'Installing for development section', but I agree we should provide more details on installing for prod.